scripts/add_identifiers_to_existing_preprints.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import time
+
+from website.app import init_app
+from website.identifiers.utils import get_or_create_identifiers, get_top_level_domain
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def add_identifiers_to_preprints():
+    from osf.models import PreprintService
+
+    preprints_without_identifiers = PreprintService.objects.filter(identifiers__isnull=True)
```

Comment:
Will the API still return preprints successfully before and during this migration?